config.json

```diff
@@ -346,6 +346,12 @@
       ]
     },
     {
+      "slug": "spiral-matrix",
+      "difficulty": 5,
```

Comment:
Then I agree on both the difficulty and the position according to fun factor. :)

`bank-account` is another matter:  I was never too happy with its current form as it enforces a non-FP solution (although one should probably also know how to code such side effects in Haskell, too). But that is another discussion.